tests/cockpit-tests

```diff
@@ -26,6 +26,15 @@ if [ -n "$TMPDIR" ]; then
     ln -snf $TMPDIR/libvirt /tmp/libvirt
 fi
 
+# HACK: In case of read-only /sys avoid libvirt writing there
+mount -o bind /mnt /sys/devices/virtual/net/
```

Comment:
This seems a bit dangerous when running on someone's production machine. Does libvirt insist on a writable directory? If so, the comment  is confusing, as this would *allow* libvirt to write there instead of  avoiding it.